src/homepage/PrototypesPage.js

```diff
@@ -10,6 +10,20 @@ import type {Assets} from "../webutil/assets";
 export default function makePrototypesPage(
   registry: RepoIdRegistry
 ): ComponentType<{|+assets: Assets|}> {
+  const listRegistry = (
+    <ul>
+      {registry.map((x) => (
+        <li key={stringify(x)}>
+          <Link to={`/prototypes/${x.repoId.owner}/${x.repoId.name}/`}>
+            {`${x.repoId.owner}/${x.repoId.name}`}
+          </Link>
+        </li>
+      ))}
+    </ul>
+  );
+  const displayWarning = (
+      <p>You haven't loaded any data into the registry yet!</p>
```

Comment:
So... the use of the single tick freaks the GitHub syntax higlighter out. Channeling @wchargin here, I think we should fix it by using the unicode apostrophe symbol `'` (https://www.fileformat.info/info/unicode/char/27/index.htm)

Reply:
The Unicode U+0027 APOSTROPHE _is_ the single tick that you find on your keyboard. You want U+2019 RIGHT SINGLE QUOTATION MARK `’` (Vim: `<C-k>'9`).

But yes, use that.

Reply:
You can always visit <http://rsquo.net> if you forget it. :-)